File: server/hotrod/src/main/scala/org/infinispan/server/hotrod/ServerMetadataFileFinder.scala
package org.infinispan.server.hotrod

import org.infinispan.factories.components.ModuleMetadataFileFinder

/**
 * @author gustavonalle
 * @since 8.0
 */
class ServerMetadataFileFinder extends ModuleMetadataFileFinder {
   def getMetadataFilename = "infinispan-server-hotrod-component-metadata.dat"
}

File: server/hotrod/src/main/java/org/infinispan/server/hotrod/ServerMetadataFileFinder.java
package org.infinispan.server.hotrod;

import org.infinispan.factories.components.ModuleMetadataFileFinder;
import org.infinispan.lifecycle.ModuleLifecycle;
import org.kohsuke.MetaInfServices;

/**
 * @author gustavonalle
 * @since 8.0
 */
@MetaInfServices(ModuleLifecycle.class)
public final class ServerMetadataFileFinder implements ModuleMetadataFileFinder {
   @Override
   public String getMetadataFilename() {
      return "infinispan-server-hotrod-component-metadata.dat";
   }
}
